app/router.ts

```diff
@@ -96,6 +97,15 @@ Router.map(function() {
         this.route('drafts', { path: '/drafts/:draftId' }, function() {
             this.route('register');
         });
+        this.route('addons', function() {
+            this.route('index', { path: '/' });
+            this.route('addon', { path: '/:addonId' }, function() {
+                this.route('terms');
+                this.route('account');
+                this.route('confirm');
+                this.route('configure');
+            });
```

Comment:
I think these routes can be removed since we are just doing everything on the addons index route